docs/source/contents/faqs/index.md

```diff
@@ -23,3 +23,11 @@ This depends on your use case. V2 APIs are presently alpha so not yet at GA so m
   * Simpler single model usage
   * V2 Protocol
 
+## Can I do payload logging in Seldon Core v2?
+
+By default, the input and output of every step in a pipeline (as well as the pipeline itself) is logged in Kafka.
+From there it's up to you what to do with the data.
+You could use something like [kafka connect](https://docs.confluent.io/platform/current/connect/index.html) to stream the logs to a datastore of your choice. 
```

Comment:
```suggestion
You could use something like [Kafka Connect](https://docs.confluent.io/platform/current/connect/index.html) to stream the logs to a datastore of your choice. 
```
Think this suggestion got lost because it was incompatible with the prior change (splitting over lines)